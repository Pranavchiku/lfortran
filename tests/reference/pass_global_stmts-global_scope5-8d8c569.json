{
    "basename": "pass_global_stmts-global_scope5-8d8c569",
    "cmd": "lfortran --pass=global_stmts --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope5.f90",
    "infile_hash": "63e512d3fe5d83aa6195002b9c31ddc77e1a93ac37df38989f782ed7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "pass_global_stmts-global_scope5-8d8c569.stdout",
    "stdout_hash": "71970af8e44967fa8e162c11cdbdd21b4482191049ee1a5ebc0e21ea",
    "stderr": "pass_global_stmts-global_scope5-8d8c569.stderr",
    "stderr_hash": "277d32a4e492c8aa2c65ff25fa5fc75fc7f7acec362c5de43351f5d9",
    "returncode": 0
}