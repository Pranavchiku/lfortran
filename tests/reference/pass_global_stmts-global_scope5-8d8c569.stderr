warning: Implied save feature
 --> tests/global_scope5.f90:1:1
  |
1 | integer :: x
  | ^^^^^^^^^^^^ Assuming save attribute implicitly for variables
